solidity/test/unit/Keep3rSponsor.t.sol

```diff
@@ -125,68 +126,6 @@ contract UnitKeep3rSponsorGetSponsoredJob is Keep3rSponsorUnitTest {
   }
 }
 
-contract UnitKeep3rSponsorChangeOwner is Keep3rSponsorUnitTest {
```

Comment:
if we refactor some code, we would expect the tests to stay the same, and these functionalities also be present on the contract, as well as being tested here, am i missing sth?

Reply:
how about we keep these expectations, but initialize them once for every contract we expect to have Ownable functionalities? 
then instead of testing `Ownable`, we're actually testing `AutomationVault` (for Ownable methods)

Reply:
I think it's good, instead of testing ownable as a contract, we test the methods inside the contracts that implement it.

Reply:
ok, as long as every `onlyOwner` modifier is being checked in this contract i'm ok.